global.json

```
@@ -7,6 +7,6 @@
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19474.3",
-    "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19215.12"
+    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.19524.2"
```

Are we supposed to use 5.0 version in 3.0 branch? Arcade is pushing out fixes from a servicing branch to .NET Core 3.x branches/channels.

---

@MattGal does Helix.sdk version number associated with dotnet core release? Should we just use the latest or the matching release version?

---

```suggestion
    "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19474.3"
```

This is from the same arcade build that we already take. Cop